src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -110,12 +110,12 @@ public override Task<bool> HandleRequestAsync()
             var remoteSignOutContext = new RemoteSignOutContext(Context, Scheme, Options, message);
             await Events.RemoteSignOut(remoteSignOutContext);
 
-            if (remoteSignOutContext.HandledResponse)
+            if (remoteSignOutContext.State == EventResultState.HandleResponse)
             {
                 Logger.RemoteSignOutHandledResponse();
                 return true;
             }
-            if (remoteSignOutContext.Skipped)
+            if (remoteSignOutContext.State == EventResultState.SkipToNextMiddleware)
```

Comment:
Oh I think I see the problem, I missed the fact that we were now pushing these concepts onto all of our middleware, today only OIDC + JwtBearer expose Control contexts...  yeah so we don't want to spread this out.  We certainly don't need to try and support these additional flows for 2.0 since you can't do these things with OAuth providers today.

Can we keep the control flow stuff in the BaseOpenIdConnect/BaseJwtBearer events so we are free to revisit this in 2.1?